### PR TITLE
Gives a mechanical nudge to Security to look spiffy

### PR DESCRIPTION
**Pull request by bloons3**
## About The Pull Request
Ports https://github.com/BeeStation/BeeStation-Hornet/pull/3684
- gives Security a (minor) benefit for wearing their issued gloves
- gives Security a (minor) debuff for wearing insulated gloves
- gives Assistants a (minor) buff for getting their tider outfit together
The mood changes are +/- 1, so nothing major.

## Why It's Good For The Game

This is a slight mechanical nudge for Security players to not immediately rush then wear insulated gloves all round.

## Changelog
:cl:
add: Security gets a mood buff for looking spiffy
add: Assistants get a buff for
/:cl:

### CODE REVIEW

**Comment by rdragan** [comment]
I just color the insuls black.

**Comment by Mothblocks** [comment]
image


he died

**Review by Mothblocks — CHANGES_REQUESTED** [review]
i am NOT giving assistants a buff for wearing insulated gloves, are you insane?

you didn't even put that at all in the "why is this good for the game"

**Comment by Mothblocks** [comment]
also actually TEST YOUR CODE before pr'ing it

File: code/datums/mood_events/generic_positive_events.dm [hunk 1]
```diff
@@ -236,3 +236,11 @@
 /datum/mood_event/kiss/add_effects(mob/beau)
 	if(beau)
 		description = "<span class='nicegreen'>[beau.name] blew a kiss at me, I must be a real catch!</span>\n"
+
+/datum/mood_event/sec_black_gloves
+	description = "<span class='nicegreen'>Black gloves look good on me.</span>\n"
```
Comment (Mothblocks) [inline code/datums/mood_events/generic_positive_events.dm:241]: A *permanent +1 mood buff* for security is dumb.

**Comment by Mothblocks** [comment]
this whole thing is ridiculous

**Comment by bloons3** [comment]
> also actually TEST YOUR CODE before pr'ing it

image
image

**Comment by Sinestia** [comment]
SLOT_GLOVES isn't a thing on tg--- oh i get it

**Comment by Mothblocks** [comment]
keep it to bee

**Comment by Krysonism** [comment]
A good pr speed closed by insuls sec.

**Comment by Mothblocks** [comment]
akshually insuls md

**Pull request closed without merging**